web/src/components/BaseQuestionnaireResponseForm/index.tsx

```diff
@@ -59,11 +57,23 @@ export interface BaseQuestionnaireResponseFormProps {
     questionItemComponents?: QuestionItemComponentMapping;
     groupItemComponent?: GroupItemComponent;
     onCancel?: () => void;
+    saveButtonTitle?: string;
     autoSave?: boolean;
+    draftSaveState?: RemoteData;
+    setDraftSaveState?: React.Dispatch<React.SetStateAction<RemoteData<any, any>>>;
```

Comment:
I think you can declare setDraftSaveState as `setDraftSaveState?: (data: RemoteData) => void;`

Reply:
or this way
```
draftSaveResponse?: RemoteData<QuestionnaireResponse>;
setDraftSaveResponse?: (data: RemoteData<QuestionnaireResponse>) => void;
```